.github/ISSUE_TEMPLATE/.common.yaml

```diff
@@ -0,0 +1,92 @@
+# This file contains common fields for all issue templates
+# Each field is identified by its ID, and are synced with the `sync.js` script
+#
+# WARNING: After updating this file, run `yarn sync-issue-templates`
+#
+
+body:
+  - id: issue-labels
+    type: checkboxes
+    attributes:
+      label: '📜 Issue Labels'
+      options:
+        - label: 'Please familiarize yourself with the issue labels used in this project: [LABELS.md](https://github.com/backstage/backstage/blob/master/LABELS.md)'
+          required: true
+  - id: search-terms
+    type: textarea
+    attributes:
+      label: '🔎 Search Terms'
+      render: plain
+      description: |
+        What search terms did you use when trying to find similar issues?
+      placeholder: |
+        List of keywords you searched for before creating this issue. Write them down here so that others can find this issue more easily and help provide feedback.
+
+        e.g. "catalog github rate limited", "http root service configurer", "scaffolder template include", "entity card title"
+    validations:
+      required: true
+  - id: project-area
+    type: dropdown
+    attributes:
+      label: '🗃️ Project Area'
+      description: Which project area is this issue most closely related to? This will help find an owner for the issue faster.
+      options:
+        - Unknown
+        - Auditor
+        - Auth
+        - Catalog
+        - CLI Tooling
+        - Core Framework
+        - Design System
+        - Documentation
+        - Events System
+        - Home
+        - Kubernetes Plugin
+        - Notifications
+        - OpenAPI Tooling
+        - Permission Framework
+        - Search
+        - Software Templates
+        - TechDocs
+    validations:
+      required: true
+  - type: dropdown
+    id: integration
+    attributes:
+      label: '🔗 External Integration'
+      description: Is this issue related to an integration with any particular external system?
+      options:
+        - N/A
```

Comment:
Do we need both N/A and Other? 🤔

Reply:
N/A is the default and is intended to signal that this issue isn't related to any particular integration, while Other is for when it is related to some integration, but not listed. I skipped a few rare ones, but if there's a lot of use of Other it indicates that we might need to update the list